src/Jobs/GenerateAnnotationPatch.php

```diff
@@ -369,7 +369,7 @@ protected function getSVGAnnotation($shapeId, $points)
 
         switch ($shapeId) {
             case Shape::pointId():
-                $radius = 3;
+                $radius = 1;
```

Comment:
Why change this?

Reply:
I thought this was a remnant of my point-circle distinction, but I forgot that it was also used to display points with the right size. I reverted it.